### PR TITLE
Add toggle to switch readonly code editor

### PR DESCRIPTION
**Pull request by trangmei**
Fixes https://github.com/Coursemology/coursemology2/issues/3475

hide-comments-panel

### CODE REVIEW

**Comment by trangmei** [comment]
I left it to respond to window size. Only added a button for people who don't want to use narrow editor.

But yeah I see why you think it should not respond to window resizing. Having both can be annoying. I'll add some details so it's more obvious to users that the mode switching is completely manual.

**Comment by trangmei** [comment]
The UX can be annoying with this change.
no-resize

File: client/app/bundles/course/assessment/submission/components/SubmissionAnswer.jsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ class SubmissionAnswer extends Component {
     const answersLoaded = historyQuestion ? historyQuestion.pastAnswersLoaded : false;
     const isLoading = historyQuestion ? historyQuestion.isLoading : false;
     const isAutograding = questionsFlags[id] ? questionsFlags[id].isAutograding : false;
-    const disabled = noPastAnswers || isLoading || isAutograding;
+    const disabled = historyQuestion === undefined || noPastAnswers || isLoading || isAutograding;
```
Comment (jeremyyap) [inline client/app/bundles/course/assessment/submission/components/SubmissionAnswer.jsx]: `noPastAnswers` should have covered this condition. The actual bug is a few lines above

```
 const noPastAnswers = historyQuestion ? historyQuestion.answerIds.length === 0 : false
```

If there is no `historyQuestion`, then `noPastAnswers` should be `true`

File: client/app/bundles/course/assessment/submission/translations.js [hunk 2]
```diff
@@ -342,7 +342,7 @@ const translations = defineMessages({
   },
   hideCommentsPanel: {
     id: 'course.assessment.submission.readOnlyEditor.hideCommentsPanel',
-    defaultMessage: 'Hide comments panel',
+    defaultMessage: 'Show comments panel',
```
Comment (jeremyyap) [inline client/app/bundles/course/assessment/submission/translations.js]: Should change the name of the translation string as well

**Pull request merged**